### ISSUE TITLE
Assignment 12

### ISSUE THREAD

**Issue by Allengoyne** [theironyard-frontend-nashville/assignments#99]
https://medium.com/@allengoyne/backbone-js-3541e4bd90e2

**Comment by egdelwonk** [comment]
> That framework is only a useful term to begin to understand how Backbone works

Replace the word framework with pattern in this sentence.

**Comment by egdelwonk** [comment]
> Controller portion, that which manages user input, is not a part of Backbone. Clear as mud?

The views in backbone handle user input, but you are right, Backbone does not have a "Controller" class type.

**Comment by egdelwonk** [comment]
> Constructors are comprised of Model, View and Collection. 

These are some of the Objects that are available on the Backbone object, they happen to be Constructors, but are not necessarily referred to as such.  I'd suggest rewording this to something like 

> "Backbone is divided into a few key parts: Models, Views, and Collections."

**Comment by egdelwonk** [comment]
>  create objects that encapsulate date and rules for how

Typo in this paragraph, change `date` to `data`

**Comment by egdelwonk** [comment]
> Initialize is a special input parameter that can be used to define any of the three class types; Model, View or Collection.

Initialize is a function that is executed anytime any new objects of type Model, View, or Collection is instantiated, but it does not create or define any types of objects.

**Comment by egdelwonk** [comment]
> Views are instantiated by el or $el and can be looke at as fancy versions of jQuery objects.

`el` and `$el` reference the DOM element and a jQuery wrapped version of the DOM element respectively. 
They are properties on a view object that are already instantiated, so they themselves cannot be instantiate anything.

`el` is referencing a plain old vanilla DOM element (like the one you'd get if you used `document.querySelector('.class')`, `$el` is a jQuery wrapped version of that same element — that gives you access to jQuery methods on that selector. For example `this.$el.html()` to set or get the inner html of the current view's element.
